users-folder/usersModel.js

```diff
@@ -0,0 +1,34 @@
+const db = require("../database/dbConfig.js");
+
+module.exports = {
+  add,
+  find,
+  findByUsername,
+  findById,
+  findByDepartment
+};
+
+function find() {
+  return db("users").select("id", "username", "password");
```

Comment:
Why use `select` here? When you call `db('users')`, it already selects all of the fields by default:
```javascript
  return db('users');
```
except this is excluding the department now; is that on purpose?

Reply:
No this was just a slip, thank you for the suggestion!